omnibus/config/projects/dogstatsd.rb

```diff
@@ -64,6 +64,43 @@
   end
 end
 
+
+# Windows .msi specific flags
+package :zip do
+  skip_packager true
+end
+
+package :msi do
+
+  # For a consistent package management, please NEVER change this code
+  arch = "x64"
+  if windows_arch_i386?
+    upgrade_code 'a8c5b8ae-ac27-4d66-b63f-edba0e5ea477'
+    arch = "x86"
+  else
+    upgrade_code 'dd60e9df-487b-415c-ba2f-dba19ddc7ebd'
+  end
+  wix_candle_extension 'WixUtilExtension'
+  wix_light_extension 'WixUtilExtension'
+  #extra_package_dir "#{Omnibus::Config.source_dir()}\\etc\\datadog-agent\\extra_package_files"
```

Comment:
It's commented out, but let's make it consistent with how it's named in other files:
```suggestion
  #extra_package_dir "#{Omnibus::Config.source_dir()}\\etc\\datadog-dogstatsd\\extra_package_files"
```